run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+red="$(tput setaf 1)"
+green="$(tput setaf 2)"
+reset="$(tput sgr0)"
+
+failed=0
+for jsonnet_test in $(ls tests/*.jsonnet); do
+  echo "${green}Running ${jsonnet_test}...${reset}"
+  ./jsonnet \
+    --code-file "tests=${jsonnet_test}" \
+    --exec \
+    --string \
+    'local testing = import "stdlib/testing.jsonnet"; testing.run(std.extVar("tests"))' \
```

Comment:
You could also --exec 'import "'"${jsonnet_test}"'"'
That would fix the filename thing.